docs/src/main/sphinx/connector/hive.rst

```diff
@@ -375,11 +381,12 @@ Property Name                                      Description
                                                    is used for write operations. The ``${USER}`` placeholder
                                                    can be used to use a different location for each user.
 
-``hive.translate-hive-views``                      Enable translation for Hive views. (experimental)            ``false``
+``hive.translate-hive-views``                      Enable translation for :ref:`Hive views <hive-views>`.       ``false``
 
-``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate Hive views. You can    ``false``
-                                                   alternatively set the ``legacy_hive_view_translation``
-                                                   session property to ``true``.
+``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate                        ``false``
+                                                   :ref:`Hive views <hive-views>`. You can use the
+                                                   ``legacy_hive_view_translation`` catalog session property
+                                                   for temporary, catalog specific use.
```

Comment:
> catalog specific

everything here is for catalog specific.. did you mean "session specific"?

Reply:
no .. I meant catalog specific .. users seems to forget that a catalog session property is per catalog and per session. Esp when we just use the name session property and imply catalog from context it is completely missed .. this misunderstanding causes the issue btw..